src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage_account_scenarios.py

```diff
@@ -186,3 +186,109 @@ def test_create_account_sas(self, storage_account):
     def test_list_locations(self):
         self.cmd('az account list-locations',
                  checks=[JMESPathCheck("[?name=='westus'].displayName | [0]", 'West US')])
+
+
+@api_version_constraint(ResourceType.MGMT_STORAGE, max_api='2016-01-01')
+class StorageAccountTestsForStack(StorageScenarioMixin, ScenarioTest):
+    @ResourceGroupPreparer(parameter_name_for_location='location', name_prefix='cli_test_storage_stack_scenario', location='local', dev_setting_location='local')
```

Comment:
Lots of stack specific code in here.
Why?
Apart from location, shouldn't the commands work on public Azure as well?

Reply:
I think it should work as such against azure as well apart from location, Shall we take up parameterizing or adding a condition as a separate PR without blocking this one?

Reply:
Would like to see it parameterized as there shouldn't be anything specific to Stack here.
It's okay to do this in another PR.

Reply:
Can you create an issue for this so it's noted to take a look at next?

Reply:
Yes, created an issue to track this: https://github.com/Azure/azure-cli/issues/4665